lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1130,12 +1130,11 @@ def determine_agent_level_monitored_resource_labels(type)
 
       # GKE container.
       when GKE_CONSTANTS[:resource_type]
-        raw_kube_env = fetch_gce_metadata('instance/attributes/kube-env')
```

Comment:
The agent also uses the presence of the `kube-env` attribute to detect that it's running on GKE. You'll need to change line 89 to look for something else that is guaranteed to be present only in GKE.

Reply:
`metadata_attributes: %w(cluster-name cluster-location)` might be the best bet as a replacement for https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/blob/a0fa440dc6a2fe96dba732f688445cff7ae4c1f5/lib/fluent/plugin/out_google_cloud.rb#L89.

![image](https://user-images.githubusercontent.com/5287526/44613116-4b6f5a80-a7dd-11e8-9b91-841ee4debd80.png)

Reply:
asdf

Reply:
Yes, what @qingling128 says seems correct. Can we please make that change and remove any mention of `kube-env` from this code?
Will metadata concealment also hide the fact that `instance/attributes/kube-env` is present? If yes, we definitely need to change this. Otherwise it's more of a cleanup.